### ISSUE TITLE
Bad example for `<empty/>` element

### ISSUE THREAD

**Issue by martindholmes** [TEIC/TEI#2177]
Stylesheets group, looking at a ticket on processing `<empty/>`, noted that this example:

https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-empty.html#index-egXML-d53e158213

is a bit strange; the same content model could be much more concisely expressed as simply `<elementRef key="hi" minOccurs="0" maxOccurs="unbounded">`.

**Comment by lb42** [comment]
This example is discussed on another ticket I think. Its not necessarily faulty, in my view.

**Comment by sydb** [comment]
I do not think it is _faulty_ either, but I do think it is not the sort of content model one who is writing an ODD by hand would ever use, and thus that it may not be a particularly good practice to exemplify in the tagdoc. That said, there is not much _to_ exemplify, as `<empty>` is only allowed in `<content>`, `<alternate>`, and `<sequence>`.

**Comment by martindholmes** [comment]
I did read that discussion, and I see that there are some obscure cases where this might be a reasonable thing to do, but I think as the main example for the use of `<empty/>` it's pretty confusing.

**Comment by sydb** [comment]
Where is that other discussion, @martindholmes or @lb42 ?

**Comment by martindholmes** [comment]
And @sydb I still don't understand what `<empty/>` would be doing in `<sequence>`.

**Comment by martindholmes** [comment]
@sydb https://github.com/TEIC/TEI/issues/1596

**Comment by sydb** [comment]
Thank you!

**Comment by sydb** [comment]
Same thing, I presume (without thinking about it much) that it would be doing in `<alternate>` — being useless, except as a way to make life easier for automatic generation of PureODD schemas.

**Comment by martindholmes** [comment]
So if I do this:
~~~~~
<content>
<sequence>
<elementRef ident="hi"/>
<empty/>
</sequence>
</content>
~~~~~

I get around the constraint that prevents a sequence with only one item. Is that the sort of thing you're thinking of? I just can't actually imagine a scenario where some process would generate an output like this.

**Comment by sydb** [comment]
Well, I have not really thought it through, but I was toying with more like a process that reads a FileMaker Pro constraint file (or whatever) and writes out PureODD. It starts _every_ element content model by spitting out `<content><sequence><empty/></sequence></content>` and then filling in element references as needed.

**Comment by martindholmes** [comment]
Wouldn't it clean up after itself by removing empty elements from anywhere they're not sensible?

**Comment by sydb** [comment]
Even if it did, aren’t you the one who would want intermediate stages to be valid? :-)

**Comment by joeytakeda** [comment]
F2F Subgroup (@hennyu , @torstenroeder , and @joeytakeda ) agree that, while this isn't an _incorrect_ example, it is not the best for exemplifying the use of `<empty>`. We propose removing this example and adding an example with a bit more context (i.e. showing the content model of `milestone`). PR forthcoming.
